lib/galaxy/jobs/runners/kubernetes.py

```diff
@@ -435,44 +418,44 @@ def check_watched_item(self, job_state):
 
             # This assumes jobs dependent on a single pod, single container
             if succeeded > 0:
-                self.__produce_log_file(job_state)
```

Comment:
Unfortunately in the new setup I fail to see the stdout and stderrr of jobs in Galaxy after execution. Any ideas @mvdbeek? I can confirm that this is viewable before this PR (as I was testing the same tool at both sides.

Reply:
(I'm not sure the cuprit is actually there, I just pointed to any potentially related piece of the code)

Reply:
Are you using a specific tool to check this? I'm a bit surprised by this since the integration test does test visibility of stdout and stderr messages

Reply:
Well I saw it on a specific tool we are developing to bring data from our archive... but I had just run the same tool and seen its stdout at the older branch, so I know that the tool generates stdout (this is a tool we are developing here: https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/blob/develop/tools/tertiary-analysis/data/retrieve-scxa.xml). I don't think is related to the tool though. I will check with another one very soon.

Reply:
I see the lack of stdout with other tools as well (haven't been able to see stdout on any tools I have tried).

Reply:
I can't reproduce this, this is working fine for me (see the red items, this is the text from stderr):
<img width="218" alt="screen shot 2018-11-23 at 17 01 50" src="https://user-images.githubusercontent.com/6804901/48952418-666b7b00-ef42-11e8-918a-69b664f1d4df.png">

As in other job runner we only produce a message on stdout/stderr if there is actually something logged, the previous implementation was always producing a message with the pod logs I think.
The pod logs for instance shouldn't appear in a tools' stdout/stderr, if necessary the admin can look at those. If the pod fails we get `More pods failed than allowed. See stdout for pods details.`

Reply:
...interesting... do you manage to see stdout? I couldn't see that either. I did the Fatal errors like you show when using python tools. So maybe we were just misusing the stderr. The logs were used to produce the stdout, but I agree is much better to be able to separate err and out (logs produced both).